### PR TITLE
App Engine Task Queue Sample.

### PR DESCRIPTION
**Pull request by jeffmendoza**
Counter app. Copied from: https://github.com/GoogleCloudPlatform/appengine-taskqueue-counter
Added tests.

### CODE REVIEW

File: appengine/taskqueue/counter/app.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+application: taskqueue-counter
```
Comment (jonparrott) [inline appengine/taskqueue/counter/app.yaml]: Remove application and version from this.

File: appengine/taskqueue/counter/app.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+version: 1
+threadsafe: true
+
+handlers:
+- url: /.*
+  script: main.app
+
+libraries:
+- name: jinja2
+  version: latest
```
Comment (jonparrott) [inline appengine/taskqueue/counter/app.yaml]: please use an explicit version here.

File: appengine/taskqueue/counter/queue.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+queue:
```
Comment (jonparrott) [inline appengine/taskqueue/counter/queue.yaml]: Is this used in the doc? If not, it's not necessary.

File: appengine/taskqueue/counter/main_test.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import main
+import webtest
+from google.appengine.ext import ndb
+
+
+def test_app(testbed, run_tasks):
+    key_name = 'foo'
+    key = ndb.Key('Counter', key_name)
+    counter = key.get()
+    if counter:
```
Comment (jonparrott) [inline appengine/taskqueue/counter/main_test.py]: counter should never be set at this point in the test.

Comment (jeffmendoza) [inline appengine/taskqueue/counter/main_test.py]: ok, I didn't know if the local datastore persisted state, I'll refactor

File: appengine/taskqueue/counter/queue.yaml (hunk 3 above)
Comment (jeffmendoza) [inline appengine/taskqueue/counter/queue.yaml]: Looks like the first snippit in https://cloud.google.com/appengine/docs/python/config/queue uses it

Comment (jonparrott) [inline appengine/taskqueue/counter/queue.yaml]: Can you add the comment that's there here?

**Comment by jeffmendoza** [comment]
Addressed comments, fixed linter errors.

File: appengine/taskqueue/counter/app.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+api_version: 1
+threadsafe: true
+
+handlers:
+- url: /.*
+  script: main.app
+
+libraries:
+- name: jinja2
+  version: 2.8
```
Comment (jonparrott) [inline appengine/taskqueue/counter/app.yaml]: Seems like only "2.6" is available on GAE.

**Comment by jonparrott** [comment]
One last comment to address, and this is good to merge.

File: appengine/taskqueue/counter/app.yaml (hunk 5 above)
Comment (jeffmendoza) [inline appengine/taskqueue/counter/app.yaml]: Updated, is there a way to check?

**Pull request merged**